src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -32,9 +34,11 @@
     public static final String MESSAGE_SUCCESS = "Task Assigned: %1$s";
```

Comment:
GitHub doesn't allow me to comment on unchanged lines so I have to comment here, but MESSAGE_USAGE (lines 22-32) needs to be updated to reflect the extra capability to assign tasks by module code.